app-springboot/src/main/java/org/keycloak/quickstart/springboot/web/ProductController.java

```diff
@@ -36,30 +37,31 @@
 @CacheControl(policy = CachePolicy.NO_CACHE)
 public class ProductController {
```

Comment:
Why are these changes required? Is this indicating some breaking changes in our adapters?

Reply:
I'm not sure what exactly are you asking about. You added this comment to a line I didn't change. Please elaborate.

Reply:
This entire class is being completely re-written. I would like to know why as if this example worked on RH-SSO 7.3, why is a completely rewrite needed for 7.4.